decentraland/components/presentational/AccountCreationProgress.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { shallow } from "enzyme";
+import AccountCreationProgress from "./AccountCreationProgress";
+import AccountCreationStatus from "@constants/AccountCreationStatus";
+const { FUNDING_WITH_ETH } = AccountCreationStatus;
+import { anAction } from "@helpers/testHelpers";
+
+describe("AccountCreationProgress", () => {
+  it("renders the component", async () => {
```

Comment:
In the BuyLand.test.js file, we tested multiple cases. Was that overkill before? Or should we test multiple inputs for this component now that that logic lives here?

Reply:
I think we should move those test cases to this component now.

Reply:
Done.

Reply:
Okay sounds good. Thanks. Resolving this.